meetings/2024-03-13.md

```diff
@@ -0,0 +1,163 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2024-03-13T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://matrix.to/#/#solid_specification:gitter.im
+* Repository: https://github.com/solid/specification
+* Status: Published
+
+## Present
+* [Virginia Balseiro](https://virginiabalseiro.com/#me)
+* [Sarven Capadisli](https://csarven.ca/#i)
+* [Jesse Wright](https://www.jeswr.org/#me)
+* Hadrian Zbarcea (Inrupt)
+* Jeff Zucker
+* [elf Pavlik](http://elf-pavlik.hackers4peace.net)
+* Maxime Lecoq
+* April Daly
+* [Rahul Gupta](https://cxres.pages.dev/profile#i)
+* [Matthias Evering](https://solidweb.me/testpro/)
+
+### Regrets
+* Tim BL
+---
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/w3c-cg/solid/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Sarven Capadisli
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+### Special Topic Meetings
+URL: https://github.com/orgs/solid/projects/16/views/1
+* VB: re https://github.com/solid/specification/discussions/555 , Ted mentioned to deprecate so I marked it as archived. Agree that I will move what is discussed. We don't have discussed status anymore on the board. Having done may be enough. Try and see how it goes.
+* eP: We can't put all discussions into done because sometimes we decide to come back to discuss. After the meeting we know we are not "done" and need to come back at osme point. Perhaps a way to mark as in progress?
```

Comment:
```suggestion
* eP: We can't put all discussions into done because sometimes we decide to come back to discuss. After the meeting we know we are not "done" and need to come back at some point. Perhaps a way to mark as in progress?
```